functions/http/1.1/function.json

```diff
@@ -0,0 +1,119 @@
+{
+  "description": "Sends a HTTP(S) request",
+  "label": "HTTP(S)",
+  "category": "External",
+  "icon": {
+    "name": "CloudIcon",
+    "color": "Orange"
+  },
+  "options": [
+    {
+      "info": "Enter what protocol the receiving end needs to use. You can choose between HTTP and HTTPS, which one you should use depends on your host.",
+      "label": "Protocol",
+      "meta": {
+        "type": "Select",
+        "default": "https",
+        "values": [
+          {
+            "label": "HTTP",
+            "value": "http"
+          },
+          {
+            "label": "HTTPS",
+            "value": "https"
+          }
+        ]
+      },
+      "name": "protocol"
+    },
+    {
+      "info": "Map the values that you want to use in your url",
+      "label": "Url Parameters",
+      "meta": {
+        "type": "Map"
+      },
+      "name": "urlParameters"
+    },
+    {
+      "info": "Enter the URL of the API that you want to use.",
+      "label": "Url",
+      "meta": {
+        "type": "Text"
+      },
+      "name": "url"
+    },
+    {
+      "info": "HTTP defines a set of request methods to indicate the desired action to be performed for the given resource.",
+      "label": "Method",
+      "meta": {
+        "type": "Select",
+        "values": [
+          {
+            "label": "GET",
+            "value": "get"
+          },
+          {
+            "label": "POST",
+            "value": "post"
+          },
+          {
+            "label": "DELETE",
+            "value": "delete"
+          },
+          {
+            "label": "PUT",
+            "value": "put"
+          }
+        ]
+      },
+      "name": "method"
+    },
+    {
+      "info": "Sometimes you may need to pass some additional information in your request, this is possible by adding variables in your header.",
+      "label": "Headers",
+      "meta": {
+        "type": "Map"
+      },
+      "name": "headers"
+    },
+    {
+      "info": "Query parameters can be defined as the optional key-value pairs that appear after the question mark in the URL.",
+      "label": "Query Parameters",
+      "meta": {
+        "type": "Map"
+      },
+      "name": "queryParameters"
+    },
+    {
```

Comment:
Same here is this the correct order first the map then te multiline text? Maybe discuss with team or ux